### PR TITLE
Changes to align with latest VFB search tool and SOLR schema changes

### PR DESCRIPTION
**Pull request by Robbie1977**

### CODE REVIEW

**Comment by Robbie1977** [comment]
tested locally and fixes solr VFBid queries.

Awaiting build before it can be tested in nblast queries openCPU server

**Comment by jefferis** [comment]
Thanks a lot @Robbie1977! The travis build failed. Do you know if this is because ocpu is not up yet? Because we need to change some server URLs or something else? 

```
> vfb_nblast('fru-M-200266')
Error in vfb_nblast("fru-M-200266") : Bad Request (HTTP 400).
Calls: vfb_nblast -> stop_for_status
```

**Comment by Robbie1977** [comment]
it's the openCPU server which fingers crossed I'll have fixed ASAP

**Comment by Robbie1977** [comment]
@jefferis can you restart the travis test as it should now pass

**Comment by Robbie1977** [comment]
One of the two servers seems to not have the latest 2 secs...

**Comment by Robbie1977** [comment]
@jefferis everything looks good but getting issues with the openCPU server reverting back to your version of vfbr so think we will have to merge before and deeply before the tests using the VFB server will work.

**Comment by Robbie1977** [comment]
Resolves https://github.com/VirtualFlyBrain/VFB2/issues/303

**Comment by jefferis** [comment]
OK - merged, hoping that it's going to work out ...

File: R/vfb_nblast.R [hunk 1]
```diff
@@ -71,7 +71,7 @@ vfb_nblast<-function(query, n=50, target=c("FlyCircuit", "GMR-Gal4"),
     if(target=="GMR-Gal4")
       res[['id']]=gmr_vfbid(res[['id']])
     else
-      res[['id']]=vfb_tovfbids(res[['id']])
+      res[['id']]=vfb_tovfbids(res[['id']], fixed=FALSE)
```
Comment (jefferis) [inline R/vfb_nblast.R:74]: @Robbie1977 does fixed work at all now? see https://github.com/jefferis/vfbr/issues/19

Comment (jefferis) [reply R/vfb_nblast.R:74]: @Robbie1977 in #19 I change the query field. The problem with what you have done here is that the results will no longer be in the order of the input ids, which could be a big problem. `fixed=FALSE` was intended for wildcard queries which might produce different numbers of outputs from the number of supplied input ids.

**Pull request merged**